cmd/get/project_test.go

```diff
@@ -0,0 +1,52 @@
+package get
+
+import (
+	"io"
+	"testing"
+
+	"github.com/flyteorg/flytectl/cmd/config"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flytectl/cmd/testutils"
+	"github.com/flyteorg/flyteidl/clients/go/admin/mocks"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListProjectFunc(t *testing.T) {
```

Comment:
Add unit tests with single and composite filters and sort here and for other flyte entities if not already added.